resources/windows/wix/appdatafiles.wxs

```diff
@@ -10,6 +10,9 @@
     Permanent="yes" on each file/folder component.
   -->
   <Fragment>
+
+    <PropertyRef Id="WIX_ACCOUNT_ADMINISTRATORS" />
```

Comment:
@Iristyle Had that in initially, but it broke the build with a duplicate symbol type error.
I've verified that the property is being set and properly applied to the ```facts.d``` directory.